src/App.js

```diff
@@ -1,35 +1,27 @@
 import React from 'react';
-// import {
-//  createBrowserRouter, createRoutesFromElements, RouterProvider, Route, Link, Outlet,
-// } from 'react-router-dom';
-// import Categories from './components/Categories';
-// import BookStore from './components/BookStore';
+import {
+  createBrowserRouter, createRoutesFromElements, RouterProvider, Route, Link, Outlet,
+} from 'react-router-dom';
+import Categories from './components/Categories';
+import BookStore from './components/BookStore';
 import './App.css';
-import Books from './redux/features/books/Books';
-import Categories from './redux/features/categories/Categories';
 
 function App() {
-/*  const router = createBrowserRouter(
+  const router = createBrowserRouter(
     createRoutesFromElements(
       <Route path="/" element={<Root />}>
         <Route path="/bookstore" element={<BookStore />} />
         <Route path="/categories" element={<Categories />} />
```

Comment:
- [OPTIONAL] I will advise you to use the BookStore as the root route so that it can be user friendly and understanding.